src/main/java/com/o19s/es/ltr/ranker/dectree/ModelObjective.java

```diff
@@ -0,0 +1,59 @@
+package com.o19s.es.ltr.ranker.dectree;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Represents an XGBoost model learning objective
+ *
+ * Depending on the objective, the model prediction may require a transform. Currently only untransformed
+ * and logistic (sigmoid) types are implemented.
+ * See <a href="https://xgboost.readthedocs.io/en/latest/parameter.html#learning-task-parameters">task params</a>
+ */
+public class ModelObjective {
```

Comment:
While for XGBoost this is a model objective for the plugin and prediction purpose this is just a normalization of the output score.
I think it would clearer to make this a factory of normalization methods and possibly calling this Normalizers instead of ModelObjective